_rules/SC4-1-1-unique-id.md

```diff
@@ -0,0 +1,90 @@
+---
+name: unique element id attributes
+description: |
+  This rule checks that all `id` attribute values on a single page are unique.
+
+success_criterion:
+- 4.1.1 # Success Criterion 4.1.1 (Parsing)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Bryn Anderson
+- Jey Nandakumar
+- Kamyar Rasta
+- Wilco Fiers
+---
+
+## Test procedure
+
+### Applicability
+
+Each test target has an `id` attribute that is unique within the [document context](https://github.com/auto-wcag/auto-wcag/pull/110/files#insert-link-here) of the element.
```

Comment:
This was supposed to be the expectation rather than the applicability.

Reply:
`file#insert-link-here`? This doesn't look final to me...